cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -917,12 +948,14 @@ request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
 search = `ms_obj_changes_base_cat_customised("userchanged")`\
 | `ms_obj_user_change_out`\
+| eval cyences_severity = case( in(dest_user_subject, `cs_ad_important_user`), "medium", true(), "low")  \
```

Comment:
Can we add some automation to figure out user is a high priority for the user or not?
If we can not get it directly, let's create a Jira ticket for the future.

Reply:
Currently, there is no user_type field for the AD events. Also, checked the cs_ad_obj_user lookup but all user has the user_type=user value so need to find a way to achieve this. Created Jira: CY-426